### PR TITLE
Play 3465 python 3 support

### PR DESCRIPTION
**Pull request by jayfunk**
@tjensen  This is the blueque change that was moved out of the task manager.

### CODE REVIEW

File: blueque/redis_task.py [hunk 1]
```diff
@@ -25,8 +26,12 @@ def _key(*args):
 
     def get_task_data(self):
         task_data = {}
-
+        logging.info("get_task_data")
```
Comment (spiralman) [inline blueque/redis_task.py]: This debug logging should be removed.

File: setup.py [hunk 2]
```diff
@@ -6,11 +6,11 @@
 
 install_requires = [
     "argparse",
-    "redis"
+    "redis <= 2.10.6",
```
Comment (spiralman) [inline setup.py]: Would `redis < 3.0.0` be better, in the (unlikely) event that a bug-fix or security fix is released on the 2.X branch of the library?

Comment (jayfunk) [reply setup.py]: We are hitting an issue with the build server contextually bringing in redis 3.10.x which is making this restriction unusable.

**Comment by jayfunk** [comment]
@spiralman Back to you.

**Comment by shroudofthurin** [comment]
@spiralman updated setup to have redis < 3.0.0 in the install requires

**Pull request merged**